README.md

```diff
@@ -0,0 +1,129 @@
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [Hello World Project] <a name="about-project"></a>
+
+>This project is about building a responsive mobile first portfolio project with html and css.
+
+**[Mobile First Portfolio Project]** 
+
+## 🛠 Built With <a name="built-with"> HTML and CSS</a>
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Toolbar and headline section made responsive for mobile screens of 375px and 768px]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Text editor, run commands and browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+  cd my-folder
+  git clone https://github.com/AFakhriddinov/portfolio-mobile-first.git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abdul-Aziz Fakhriddinov**
+
+- GitHub: [@githubhandle](https://github.com/AFakhriddinov)
+- Twitter: [@facebookhandle](https://www.facebook.com/abdulaziz.faxriddinov)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdul-aziz-fakhriddinov-5297a6107/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+Build desktop version
+Add more content
+Add form section
+Make responsive for other mobile devices
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+If you like this project, let me know and we can improve it further
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse team and reviewers who helped me make this project look more professional
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[How can this project be imporoved]**
+
+  - [more content could be added to this website]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## Licence <a name="licence"></a>
+For this project, MIT licence is used. The link to MIT licence is https://github.com/AFakhriddinov/portfolio-mobile-first/blob/main/LICENSE
```

Comment:
- [ ] Good job adding the License file. instead of placing the link raw, consider using the markdown option to make it a clickable link just like the README did. e.g [MIT](https://github.com/AFakhriddinov/portfolio-mobile-first/blob/main/LICENSE)

## Template form
![image](https://user-images.githubusercontent.com/50990925/220831409-d49f3357-0b0d-4ac5-ba7e-74969988dab3.png)